test/io/sungeun/funcPtr6.good

```diff
@@ -1 +1 @@
-funcPtr6.chpl:5: error: FIX ME!
+funcPtr6.chpl:5: error: 'foo' cannot be captured as a value because it is a generic function
```

Comment:
What in this PR is changing the behavior of these futures?

Reply:
Nothing. I just noticed them because I was spot-checking tests using CommDiagnostics and the futures themselves were out of date (requesting to not have C compilation errors; but we have reasonable errors now IMO).

Reply:
Moved those to PR #16265.